tracer/src/Datadog.Trace/AspNet/TracingHttpModule.cs

```diff
@@ -147,6 +147,9 @@ private void OnBeginRequest(object sender, EventArgs eventArgs)
                 var security = Security.Instance;
                 if (security.Settings.Enabled)
                 {
+                    security.InstrumentationGateway.RaiseRequestStart(httpContext, httpContext.Request, scope.Span);
+                    security.InstrumentationGateway.RaiseBlockingOpportunity(httpContext, scope, tracer.Settings, (_) => { });
```

Comment:
Also wondering if the line 
`                tracer.TracerManager.Telemetry.IntegrationGeneratedSpan(IntegrationId);`
shouldn't go before the security treatment or at least the `RaiseBlockOpportunity` or in the lamda (do before blocking) because after we'll end up directly in the catch

Reply:
fixed in bcfbde6